README.md

```diff
@@ -35,6 +35,8 @@ The only required option is `orgId`, all others are optional.
 * `recordCrossDomainIFrames` - Defaults to `false`. FullStory can record cross-domain iFrames if: 1. The FullStory Browser SDK is running in the cross-domain iFrame and 2. `recordCrossDomainIFrames` is set to `true` in the cross-domain iFrame and 3. The FullStory Browser SDK is running in the parent page of the cross-domain iFrame. Click [here](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) for a detailed explanation of what "cross-domain" means. Before using, you should understand the security implications, and configure your [Content Security Policy](https://www.html5rocks.com/en/tutorials/security/content-security-policy/) (CSP) HTTP headers accordingly - specifically the frame-ancestors directive. Failure to configure your CSP headers while using this setting can bypass IFrames security protections that are included in modern browsers. More information about cross-domain iFrame recording can be found on our [Knowledge Base](https://help.fullstory.com/hc/en-us/articles/360020622514-Can-FullStory-capture-content-that-is-presented-in-iframes-#2-the-outer-page-is-running-fullstory-and-you-have-iframes-runni). Note: the `recordCrossDomainIFrames` parameter is the same as the `window['_fs_run_in_iframe']` referenced in the KB article.
 * `recordOnlyThisIFrame` - When set to `true`, this tells FullStory that the IFrame is the "root" of the recording and should be its own session; defaults to `false`. Use this when your app is embedded in an IFrame on a site not running FullStory or when the site *is* running FullStory, but you want your content sent to a different FullStory org.
 * `devMode` - Set to `true` if you want to deactivate FullStory in your development environment. When set to `true`, FullStory will shutdown recording and all subsequent SDK method calls will be no-ops. At the time `init` is called with `devMode: true`, a single `event` call will be sent to FullStory to indicate that the SDK is in `devMode`; this is to help trouble-shoot the case that the SDK was accidentally set to `devMode: true` in a production environment. Additionally, any calls to SDK methods will `console.warn` that FullStory is in `devMode`. Defaults to `false`.
+* `startCaptureManually` - Set to `true` if you want to start capture manually using `FS('start')`. FullStory will load but wait for a call to `FS('start')` to begin capturing. See [Manually Delay Data Capture](https://developer.fullstory.com/browser/v2/auto-capture/capture-data/#manually-delay-data-capture) for more information.  Defaults to `false`.
```

Comment:
unless I'm mistaken, we should remove the `v2` from this link?
```suggestion
* `startCaptureManually` - Set to `true` if you want to start capture manually using `FS('start')`. FullStory will load but wait for a call to `FS('start')` to begin capturing. See [Manually Delay Data Capture](https://developer.fullstory.com/browser/auto-capture/capture-data/#manually-delay-data-capture) for more information.  Defaults to `false`.
```

Reply:
Yes, that's a good idea. We'll have a server-side redirect in place to catch any extraneous `https://developer.fullstory.com/browser/v2/*` in the wild, but in this case it's good to fix it up in the doc.

Reply:
I don't feel super strongly here, this is using "browser api v2" so I don't mind it pointing to v2 docs (rather than always go to the latest). Given that we'd have to remember to update links on version bump..

Reply:
> so I don't mind it pointing to v2 docs

@sabrina-li I think `v2` paths will always go to latest because of the server redirects @mattgrosvenor-fs is putting in place:

`browser/v2/*` --> `browser/*`

Reply:
Right, I was thinking if/when v3 is released, the server redirect would need to be updated to 
- `browser/v3/*` --> `browser/*` === v3 docs
- While `browser/v2/*` will remain pointing at v2 docs

In that case, what page should this link go to?

Reply:
- browser/v3/* --> browser/* === v3 docs
- While browser/v2/* will remain pointing at v2 docs

Yes we'd need to update the redirects so that `v2` points to the newly deprecated V2 API docs.

This should be fine, because:

- There likely won't be many pre-release V2 doc URLs out in the wild anyway
- We'll have a "deprecated" banner on V2 docs to get folks to the current version if they come in from a direct link on a `v2` URL.

Hope that makes sense!

Reply:
SGTM

Reply:
Hi @sabrina-li ! Thanks for the review.

I am now reworking this PR to be the "final" (🤞 ) beta release for api v2. A couple of questions:
- It probably makes sense to keep this link as v2/ for now? (since there's no anchor at just `browser/auto-capture/capture-data/#manually-delay-data-capture`)
- I bumped the package.json to `2.0.0.beta.3` to match the beta tag of `@fullstory/snippet`. Does this make sense? Should this happen _after_ the merge as part of the release process?